apps/designer/app/shared/design-system/stitches.config.ts

```diff
@@ -497,7 +497,7 @@ export const darkTheme = createTheme("dark-theme", {
     muted: "$slate8",
     primary: "$blue9",
     hiContrast: "$slate12",
-    loContrast: "$slate1",
+    loContrast: "hsl(0 0% 17%)",
```

Comment:
we can't express this color with radix colors? I wonder why ...

Reply:
there is nothing in radix that is very close?

Reply:
those other custom colors were a copy paste from this https://github.com/radix-ui/design-system

I think the problem with using custom colors is getting an equivalent dark/light mode color, unless it can be the same for both